app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -204,12 +205,19 @@ class GithubWebhook extends RequestHandler<Body> {
           !file.filename.startsWith('dev/bots/')) {
         needsTests = true;
       }
-      if (file.filename.endsWith('_test.dart')) {
+      if (file.filename.endsWith('_test.dart') ||
+          file.filename.endsWith('.expect') ||
+          file.filename.contains('test_fixes')) {
```

Comment:
These two could be easily supported too:
> * A PR that only affect code inside the .github directory is exempt from the need to add tests.
> * A PR that only affects .md files is exempt from the need to add tests.

Reply:
They already are. Only files ending with `.dart` are marked as `needsTests`. I just think we are all trained now to ask. 😄 
For example: The bot did not flag this `.md`-only change: https://github.com/flutter/flutter/pull/90060

Reply:
Hmm, changes to other files should also get tests, not just .dart files...

Reply:
That's what I figures, I'll add this to my list of updates once the cocoon tree is open again.